### DISCUSSION TITLE
Breaking changes!

### DISCUSSION THREAD

**Discussion by SylEleuth** [SylEleuth/gruvbox-plus-icon-pack#19] [Announcements]
Finally I've decided to separate Gruvbox Plus into dark and light theme. To even start with that I've had to create dedicated folders. If You were linking icons to .locale/share/icons directory you should link folder `Gruvbox-Plus-Dark` (and in the future `Gruvbox-Plus-Light`).
I will add light theme when it will be ready.

**Comment by Me-At-The-Age-Of-Git** [comment]
I **love** your pack, thank you for your awesome work!! :smiley: :+1:
